x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/Watcher.java

```diff
@@ -285,6 +285,13 @@ public class Watcher extends Plugin implements SystemIndexPlugin, ScriptPlugin,
      * to these old nodes that the mappings are newer than they are.
      */
     private static final String LEGACY_VERSION_FIELD_VALUE = "8.12.0";
+
```

Comment:
Not Blocking: A bit of Javadoc that describes what the script does, and why, could be helpful here.